app/views/bulk_upload_shared/guidance.html.erb

```diff
@@ -54,7 +54,14 @@
 
     <div class="govuk-!-padding-bottom-4">
       <h2 class="govuk-heading-s">Getting help</h2>
-      <p class="govuk-body">There is no step-by-step bulk upload guide like there is with single log upload. However, you can download <%= govuk_link_to "our template", @form.template_path %>, which you can copy-paste data into from your systems column-by-column. You can also view a post-upload report showing any data errors, and our <%= govuk_link_to "data specification", @form.specification_path, target: "_blank" %> can help fix these.</p>
+      <p class="govuk-body">There is no step-by-step bulk upload guide like there is with single log upload. However, you can download <%= @form.year == 2022 ? govuk_link_to("our template", @form.template_path) : "one of our templates" %>, which you can copy-paste data into from your systems column-by-column. You can also view a post-upload report showing any data errors, and our <%= govuk_link_to "data specification", @form.specification_path, target: "_blank" %> can help fix these.</p>
```

Comment:
surely they'd copy-paste row-by-row (1 log at a time) rather than column-by-column (1 attribute at a time)?

Reply:
I'll ask on the ticket, ty!